routing/dht/routing.go

```diff
@@ -84,7 +84,7 @@ func (dht *IpfsDHT) GetValue(ctx context.Context, key key.Key) ([]byte, error) {
 	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
 
-	vals, err := dht.GetValues(ctx, key, 16)
+	vals, err := dht.GetValues(ctx, key, (KValue/2)+1)
```

Comment:
- why `KValue/2` ? 
- document choice.

again, in a large network, this is totally going to be overshadowed by tons of records floating around. need a large sample to have hope of getting the latest.